AppCreationScripts/AppCreationScripts.md

```diff
@@ -0,0 +1,107 @@
+# Registering the Azure Active Directory applications and updating the configuration files for this sample using PowerShell scripts
+
+## Goal of the scripts
+
+### Presentation of the scripts
+
+This sample comes with two PowerShell scripts, which automate the creation of the Azure Active Directory applications, and the configuration of the code for this sample. Once you run them, you will only need to build the solution and you are good to test.
+
+These scripts are:
+
+- `Configure.ps1` which:
+  - creates Azure AD applications and their related objects (permissions, dependencies, secrets),
+  - changes the configuration files in the C# and JavaScript projects.
+  - creates a summary file named `createdApps.html` in the folder from which you ran the script, and containing, for each Azure AD application it created:
+    - the identifier of the application
+    - the AppId of the application
+    - the url of its registration in the [Azure portal](https://portal.azure.com).
+
+- `Cleanup.ps1` which cleans-up the Azure AD objects created by `Configure.ps1`. Note that this script does not revert the changes done in the configuration files, though. You will need to undo the change from source control (from Visual Studio, or from the command line using, for instance, git reset).
+
+### Usage pattern for tests and DevOps scenarios
+
+The `Configure.ps1` will stop if it tries to create an Azure AD application which already exists in the tenant. For this, if you are using the script to try/test the sample, or in DevOps scenarios, you might want to run `Cleanup.ps1` just before `Configure.ps1`. This is what is shown in the steps below.
+
+## How to use the app creation scripts ?
+
+### Pre-requisites
+
+To use the app creation scripts:
+
+1. Open PowerShell (On Windows, press  `Windows-R` and type `PowerShell` in the search window)
+2. Navigate to the root directory of the project.
+3. Until you change it, the default Execution Policy for scripts is usually `Restricted`. In order to run the PowerShell script you need to set the Execution Policy to `Unrestricted`. You can set this just for the current PowerShell process by running the command:
+    ```PowerShell
+    Set-ExecutionPolicy -Scope Process -ExecutionPolicy Unrestricted
+    ```
+4. If you have never done it already, in the PowerShell window, install the AzureAD PowerShell modules. For this, type:
+    ```PowerShell
+    Install-Module AzureAD
+    ```
+5. Go to the `AppCreationScripts` sub-folder. From the folder where you cloned the repo,
+    ```PowerShell
+    cd AppCreationScripts
+    ```
+6. Run the scripts. See below for the [four options](#four-ways-to-run-the-script) to do that.
+7. Open the Visual Studio solution, and in the solution's context menu, choose **Set Startup Projects**.
```

Comment:
I don't think the Visual studio steps are relevant for this sample

Reply:
indeed

Reply:
fixed